dask_kubernetes/logs.py

```diff
@@ -0,0 +1,28 @@
+class Log(str):
+    """A container for logs."""
+
+    def _widget(self):
+        from ipywidgets import HTML
```

Comment:
What happens if `ipywidgets` isn't installed?

Also, do we need `ipywidgets` here?  Would implementing a `_repr_html_` method work instead?

```python
def _repr_html_(self):
    return "<pre><code>... "
```

I think that this has the advantage of also rendering nicely in static notebooks.

Perhaps that wouldn't work for the accordion though, so maybe this isn't worth it.

Reply:
> What happens if ipywidgets isn't installed?

I don't think `obj._widget` is called if ipywidgets is not importable.